src/main/java/com/google/api/generator/gapic/model/Method.java

```diff
@@ -53,9 +53,8 @@
   @Nullable
   public abstract String mixedInApiName();
 
-  // TODO(miraleung): May need to change this to MethodArgument, Field, or some new struct
   // HttpBinding pending dotted reference support.
-  public abstract List<String> httpBindings();
+  public abstract HttpRuleBindings httpBindings();
```

Comment:
Continuing the comment above; Can we put a `@Nullable` on this?

Reply:
I really would rather not have `null`s for things like this. Please check the other comments about HttpBindings. WDYT about renaming the `HttpRuleBindings` to just `HttpBindings` to make things clearer that this class is not strictly about the `google.api.http` option in proto files.